website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -173,9 +175,17 @@ const MessageActions = ({
           {t("report_action")}
         </MenuItem>
         <MenuDivider />
-        <MenuItem as="a" href={`/messages/${messageId}`} target="_blank" icon={<MessageSquare />}>
+        <MenuItem as="a" href={`/messages/${message.id}`} target="_blank" icon={<MessageSquare />}>
           {t("open_new_tab_action")}
         </MenuItem>
+        {role === "admin" && (
+          <>
+            <MenuDivider />
+            <MenuItem as="a" href={`/admin/manage_user/${message.user_id}`} target="_blank" icon={<Edit />}>
```

Comment:
Maybe we should you the user or eye icon here?

Reply:
Agreed, done.